dask/array/einsumfuncs.py

```diff
@@ -236,7 +236,7 @@ def einsum(*operands, dtype=None, optimize=False, split_every=None, **kwargs):
 
     if len(inputs) > 1 and len(outputs) > 0:
         # Calculate the increase in chunk size compared to the largest input chunk
-        max_chunk_sizes, max_chunk_size_inuput = [], 0
+        max_chunk_sizes, max_chunk_size_inuput = [], 1
```

Comment:
nit (it looks like this typo is already on main)
```suggestion
        max_chunk_sizes, max_chunk_size_input = [], 1
```